LiskHQ/lisk-mobile
# Implement the basic structure for the localization - Closes #421

# What was the bug or feature?
Described in #421 

### How did I fix it?
Used [react.i18next](https://react.i18next.com)

## Type of change
- Enhancement (a non-breaking change which adds functionality)

### How to test it?
 - Should change the language using the language selector in the settings page.
 - Should default to the language of the device.
 - Should store the selected language for the next run.

# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
